README.md

```diff
@@ -56,14 +56,6 @@ Guides can be found [here](https://www.starknetjs.com/guides/intro)
 
 [Click Here](https://www.starknetjs.com/docs/API/)
 
-## 🚀 Powered by Starknet.js
-
-- [Argent X - the first StarkNet wallet](https://github.com/argentlabs/argent-x)
-- [Braavos - your new wallet on top of StarkNet](https://chrome.google.com/webstore/detail/braavos-wallet/jnlgamecbpmbajjfhmmmlhejkemejdma)
-- [React + Starknet.js boilerplate](https://github.com/fracek/starknet-react-example)
-- [AMM Demo](https://www.starknetswap.com/)
-- [mySwap - the first DeFi app to launch on StarkNet](https://myswap.xyz)
-
```

Comment:
realistically almost everything in JS world interacting with Starknet is using starknet.js :D